--- gpm-1.20.6/src/lib/report-lib.c~	2012-12-12 13:45:18.744097221 +0100
+++ gpm-1.20.6/src/lib/report-lib.c	2012-12-12 13:45:49.834096126 +0100
@@ -47,7 +47,7 @@
                            log_level = LOG_CRIT; break;
    }
 #ifdef HAVE_VSYSLOG
-   syslog(log_level, string);
+   syslog(log_level, "%s", string);
    vsyslog(log_level, text, ap);
 #else
    fprintf(stderr,"%s[%s(%d)]:\n",string,file,line);
