diff -urNp -x '*.orig' gpm-1.20.7.org/src/lib/report-lib.c gpm-1.20.7/src/lib/report-lib.c
--- gpm-1.20.7.org/src/lib/report-lib.c	2012-10-26 23:21:38.000000000 +0200
+++ gpm-1.20.7/src/lib/report-lib.c	2023-12-02 17:21:05.021705751 +0100
@@ -36,13 +36,13 @@ void gpm_report(int line, char *file, in
 
    switch(stat) {
       case GPM_STAT_INFO : string = GPM_TEXT_INFO ;
-                           log_level = LOG_INFO; break;
+                           log_level = LOG_INFO; return;
       case GPM_STAT_WARN : string = GPM_TEXT_WARN ;
                            log_level = LOG_WARNING; break;
       case GPM_STAT_ERR  : string = GPM_TEXT_ERR  ;
                            log_level = LOG_ERR; break;
       case GPM_STAT_DEBUG: string = GPM_TEXT_DEBUG;
-                           log_level = LOG_DEBUG; break;
+                           log_level = LOG_DEBUG; return;
       case GPM_STAT_OOPS : string = GPM_TEXT_OOPS;
                            log_level = LOG_CRIT; break;
    }
diff -urNp -x '*.orig' gpm-1.20.7.org/src/report.c gpm-1.20.7/src/report.c
--- gpm-1.20.7.org/src/report.c	2012-10-26 23:21:38.000000000 +0200
+++ gpm-1.20.7/src/report.c	2023-12-02 17:21:05.021705751 +0100
@@ -138,6 +138,7 @@ void gpm_report(int line, char *file, in
       case GPM_RUN_DAEMON:
          switch(stat) {
             case GPM_STAT_INFO:
+	       break;
 #ifdef HAVE_VSYSLOG
                syslog(LOG_INFO | LOG_USER, GPM_STRING_INFO);
                vsyslog(LOG_INFO | LOG_USER, text, ap2);
