--- gpm-cvs/src/liblow.c~	Sun Oct  6 10:36:25 2002
+++ gpm-cvs/src/liblow.c	Sun Oct  6 10:36:25 2002
@@ -258,7 +258,7 @@
          /* do we really need this check ? */
          if(strncmp(tty,consolename,strlen(consolename)-1)
             || !isdigit(tty[strlen(consolename)-1])) {
-            gpm_report(GPM_PR_ERR,"strncmp/isdigit/consolename failed");
+           /* gpm_report(GPM_PR_ERR,"strncmp/isdigit/consolename failed"); */
             goto err;
          }
           
@@ -367,7 +367,7 @@
 
   /*....................................... Error: free all memory */
    err:
-   gpm_report(GPM_PR_ERR,"Oh, oh, it's an error! possibly I die! ");
+  /* gpm_report(GPM_PR_ERR,"Oh, oh, it's an error! possibly I die! "); */
    do {
       new=gpm_stack->next;
       free(gpm_stack);
