--- ./doc/Makefile.in.org	Sun Jun 30 17:56:12 2002
+++ ./doc/Makefile.in	Sun Jun 30 17:58:07 2002
@@ -113,29 +113,21 @@
 # install & uninstall
 
 installdirs:	
-	$(MKDIR) $(man1dir) $(man7dir) $(man8dir) $(infodir)	
+	$(MKDIR) $(DESTDIR)$(man1dir) $(DESTDIR)$(man7dir) $(DESTDIR)$(man8dir) $(DESTDIR)$(infodir)	
 
 install:	all installdirs
-	$(INSTALL_DATA) -m 644 mev.1          $(man1dir)
-	$(INSTALL_DATA) -m 644 mouse-test.1   $(man1dir)
-	$(INSTALL_DATA) -m 644 gpm-root.1     $(man1dir)
-	$(INSTALL_DATA) -m 644 gpm-types.7    $(man7dir)
-	$(INSTALL_DATA) -m 644 gpm.8          $(man8dir)
-	$(INSTALL_DATA) -m 644 $(srcdir)/gpm.info       $(infodir)
+	$(INSTALL_DATA) -m 644 mev.1          $(DESTDIR)$(man1dir)
+	$(INSTALL_DATA) -m 644 mouse-test.1   $(DESTDIR)$(man1dir)
+	$(INSTALL_DATA) -m 644 gpm-root.1     $(DESTDIR)$(man1dir)
+	$(INSTALL_DATA) -m 644 gpm-types.7    $(DESTDIR)$(man7dir)
+	$(INSTALL_DATA) -m 644 gpm.8          $(DESTDIR)$(man8dir)
+	$(INSTALL_DATA) -m 644 $(srcdir)/gpm.info       $(DESTDIR)$(infodir)
 	# Use install-info if available
 	-if $(SHELL) -c 'install-info --version' >/dev/null 2>&1; then \
 	  if [ -f $(infodir)/dir ] ; then \
-		install-info --dir-file=$(infodir)/dir $(infodir)/gpm.info; \
+		install-info --dir-file=$(infodir)/dir $(DESTDIR)$(infodir)/gpm.info; \
 	  fi; \
 	fi
-	#itz Sat Dec 26 23:21:05 PST 1998
-	#i keep all my infopages compressed and i'm tired to do it by
-	#hand, so check if there are any compressed pages and do this
-	#one too
-	-ls $(infodir)/*[-.]info.gz >/dev/null 2>&1 \
-		&& gzip -f $(infodir)/gpm.info
-	# Hmm.... shouldn't man pages be compressed too?
-   # maybe they should, but at least at my system they are not.
 
 # This potentially leaves around cached manpages,
 # e.g. /var/cache/man/cat1/mev.1.gz.  Deleting these is not our job,
--- ./contrib/Makefile.in.org	Sat May 11 18:29:14 2002
+++ ./contrib/Makefile.in	Sun Jun 30 17:51:10 2002
@@ -13,7 +13,7 @@
 
 install: all
 	if [ -n "$(ELISP)" ]; then for i in `echo $(ELISP)`; do \
-		$(INSTALL_DATA) $(srcdir)/$$i $(lispdir)/`basename $$i` ;\
+		$(INSTALL_DATA) $(srcdir)/$$i $(DESTDIR)$(lispdir)/`basename $$i` ;\
 	done; fi
 
 %.elc: %.el
--- ./src/Makefile.in.org	Mon May 20 23:58:51 2002
+++ ./src/Makefile.in	Sun Jun 30 17:54:52 2002
@@ -101,21 +101,21 @@
 installdirs:
 
 install:	check 
-	$(INSTALL_PROGRAM) gpm $(sbindir)/gpm
-	$(INSTALL_DATA) -m 644 libgpm.a $(libdir)/libgpm.a
-	$(INSTALL_DATA) -m 644 $(srcdir)/headers/gpm.h $(includedir)/gpm.h
+	$(INSTALL_PROGRAM) gpm $(DESTDIR)$(sbindir)/gpm
+	$(INSTALL_DATA) -m 644 libgpm.a $(DESTDIR)$(libdir)/libgpm.a
+	$(INSTALL_DATA) -m 644 $(srcdir)/headers/gpm.h $(DESTDIR)$(includedir)/gpm.h
 	for i in mev gpm-root; do \
-		$(INSTALL_PROGRAM) $$i $(bindir)/$$i ;\
+		$(INSTALL_PROGRAM) $$i $(DESTDIR)$(bindir)/$$i ;\
 	done
-	$(INSTALL_PROGRAM) disable-paste $(bindir)/disable-paste
+	$(INSTALL_PROGRAM) disable-paste $(DESTDIR)$(bindir)/disable-paste
 	# POSIX requires the range of a for loop be nonempty and Bash
 	# 2.x goes along; unfortunately that means an additional
 	# headache in cases like this
 	if test "x@SHLIB@" != "x" ; then \
-		$(INSTALL_DATA) -m 644 libgpm.so.@abi_full@ $(libdir)/libgpm.so.@abi_full@ ;\
-		cd $(libdir) && $(LN_S) -f libgpm.so.@abi_full@ libgpm.so ;\
+		$(INSTALL_DATA) -m 644 libgpm.so.@abi_full@ $(DESTDIR)$(libdir)/libgpm.so.@abi_full@ ;\
+		cd $(DESTDIR)$(libdir) && $(LN_S) -f libgpm.so.@abi_full@ libgpm.so ;\
       echo "WARNING: We installed a lib, you should now call ldconfig" ; \
-      echo "f.i.: ldconfig -n -l $(libdir)/libgpm.so.@abi_full@" ;\
+      echo "f.i.: ldconfig -n -l $(DESTDIR)$(libdir)/libgpm.so.@abi_full@" ;\
       echo "Or to update everything just type ldconfig"; \
 	fi
 
--- ./Makefile.in.org	Sat Feb 23 22:19:07 2002
+++ ./Makefile.in	Sun Jun 30 18:04:38 2002
@@ -50,9 +50,9 @@
 	$(MAKE) INSTALL_PROGRAM='$(INSTALL_PROGRAM) $(STRIP)' install
 
 installdirs:
-	$(MKDIR) $(libdir) $(bindir) $(sbindir) $(includedir) $(sysconfdir); \
+	$(MKDIR) $(DESTDIR)$(libdir) $(DESTDIR)$(bindir) $(DESTDIR)$(sbindir) $(DESTDIR)$(includedir) $(DESTDIR)$(sysconfdir); \
 	if test "x@ELISP@" != "x" ; then \
-		$(MKDIR) $(lispdir) ; \
+		$(MKDIR) $(DESTDIR)$(lispdir) ; \
 	fi
 
 ### GENERIC
