--- gpm-1.20.1/src/lib/liblow.c~	Sun Oct  6 10:36:25 2002
+++ gpm-1.20.1/src/lib/liblow.c	Sun Oct  6 10:36:25 2002
@@ -367,8 +367,8 @@
 
   /*....................................... Error: free all memory */
    err:
-   if (gpm_is_disabled < 2) /* be quiet if no gpmctl socket found */
-      gpm_report(GPM_PR_ERR,"Oh, oh, it's an error! possibly I die! ");
+   /* if (gpm_is_disabled < 2)
+      gpm_report(GPM_PR_ERR,"Oh, oh, it's an error! possibly I die! ");*/
    while(gpm_stack) {
       new=gpm_stack->next;
       free(gpm_stack);
