--- gpm-1.20.1/src/lib/report-lib.c.orig	Sat Feb 23 16:42:23 2002
+++ gpm-1.20.1/src/lib/report-lib.c	Fri May  3 20:16:02 2002
@@ -36,13 +36,13 @@
 
    switch(stat) {
       case GPM_STAT_INFO : string = GPM_TEXT_INFO ;
-                           log_level = LOG_INFO; break;
+                           log_level = LOG_INFO; return;
       case GPM_STAT_WARN : string = GPM_TEXT_WARN ;
                            log_level = LOG_WARNING; break;
       case GPM_STAT_ERR  : string = GPM_TEXT_ERR  ;
                            log_level = LOG_ERR; break;
       case GPM_STAT_DEBUG: string = GPM_TEXT_DEBUG;
-                           log_level = LOG_DEBUG; break;
+                           log_level = LOG_DEBUG; return;
       case GPM_STAT_OOPS : string = GPM_TEXT_OOPS;
                            log_level = LOG_CRIT; break;
    }
--- gpm-1.20.0/src/report.c.orig	Sat Feb 23 16:42:23 2002
+++ gpm-1.20.0/src/report.c	Fri May  3 20:12:51 2002
@@ -130,6 +130,7 @@
       case GPM_RUN_DAEMON:
          switch(stat) {
             case GPM_STAT_INFO:
+	       break;
 #ifdef HAVE_VSYSLOG
                syslog(LOG_INFO | LOG_USER, GPM_STRING_INFO);
                vsyslog(LOG_INFO | LOG_USER, text, ap);
