--- gpm-1.19.4/liblow.c.orig	Sun May 27 22:53:22 2001
+++ gpm-1.19.4/liblow.c	Sat Sep  8 12:31:56 2001
@@ -183,6 +183,33 @@
 }
 #endif /* SIGTSTP */
 
+/*
+ * check for devfs
+ */
+
+static int check_devfs( void )
+{
+
+  int fd, retval = GPM_IS_NOTHING;
+
+  /* first try the devfs device, because in the next time this will be
+   * the preferred one. If that fails, take the old console */
+   
+  /* Check for open new console */
+  if ((fd=open(GPM_DEVFS_CONSOLE, O_RDONLY)) >= 0)
+    retval = GPM_IS_DEVFS;
+  
+  /* Failed, try OLD console */
+  else if((fd=open(GPM_NO_DEVFS_CONSOLE, O_RDONLY)) >= 0)
+    retval = GPM_IS_NOT_DEVFS;
+  
+  if(fd >=0)
+    close(fd);
+
+  return retval;
+
+}
+
 /*-------------------------------------------------------------------*/
 int Gpm_Open(Gpm_Connect *conn, int flag)
 {
@@ -196,6 +223,7 @@
 
 
   /* check whether there is a devfs */
+  devfs_id = check_devfs();
   switch(devfs_id)
   {
 
