--- gpm-1.19.5/src/gpm.c.orig	Mon Sep 24 21:57:44 2001
+++ gpm-1.19.5/src/gpm.c	Mon Sep 24 22:03:08 2001
@@ -1058,8 +1058,8 @@
       oops(ctladdr.sun_path);
   maxfd=max(maxfd,ctlfd);
 
-   /* only allow the console user to access. */
-   chmod(GPM_NODE_CTL,0700);
+   /* only allow the console user to access. ** ...but WHO is console user??? */
+   chmod(GPM_NODE_CTL,0666);
 
 
 /*....................................... get screen dimensions */
