--- ./src/gpm-root.y.orgin	Sat Feb 23 16:42:23 2002
+++ ./src/gpm-root.y	Mon Mar 11 17:32:37 2002
@@ -433,9 +433,10 @@
 /*====================================================================*/
 void f__fix(struct passwd *pass)
 {
-   setgid(pass->pw_gid);
-   initgroups(pass->pw_name, pass->pw_gid);
-   setuid(pass->pw_uid);
+   if (setgid(pass->pw_gid) < 0 ||
+   	initgroups(pass->pw_name, pass->pw_gid) < 0 ||
+   	setuid(pass->pw_uid) < 0)
+   exit(1);
    setenv("HOME",    pass->pw_dir, 1);
    setenv("LOGNAME", pass->pw_name,1);
    setenv("USER",    pass->pw_name,1);
