diff -Nru gpm-1.17.9/doc/doc.gpm gpm-1.17.9.new/doc/doc.gpm
--- gpm-1.17.9/doc/doc.gpm	Fri Aug  6 07:35:33 1999
+++ gpm-1.17.9.new/doc/doc.gpm	Wed Sep  1 04:36:32 1999
@@ -34,9 +34,9 @@
 
 @ifinfo
 @format
-@dircategory Miscellaneous
+@dircategory Miscellaneous:
 @direntry
-* Gpm: (gpm).   A server wich hands mouse events to non-X programs.
+* Gpm: (gpm)				Text-mode mouse library and srv daemon
 @end direntry
 @end format
 @end ifinfo
@@ -1178,7 +1178,7 @@
 @item int gpm_hflag
 	Used to signal if a character has been generated
 	by a mouse handler. 
-        @xref{Handling Functions}
+        @xref{Handling Functions}.
 @vindex gpm_handler
 @vindex gpm_data
 @item Gpm_Handler *gpm_handler; void *gpm_data
