--- gpm-1.20.6/configure.ac.old	2009-09-25 20:21:34.000000000 +0200
+++ gpm-1.20.6/configure.ac	2009-09-25 20:24:05.000000000 +0200
@@ -113,14 +113,14 @@
 *)
     SAVELIBS=$LIBS
     LIBS=
-    for i in tinfo ncurses termcap termlib; do
+    for i in ncursesw termcap termlib; do
         if test x$LIBS = x; then
             AC_CHECK_LIB($i, tputs)
         else :; fi
     done
     TERMLIBS=$LIBS
     LIBS=
-    for i in ncurses curses; do
+    for i in ncursesw curses; do
         if test x$LIBS = x; then
             AC_CHECK_LIB($i, wgetch,,,$TERMLIBS)
         else :; fi
