--- gpm-1.20.1/Makefile.in	Sat Mar  1 11:49:41 2003
+++ gpm-1.20.1.new/Makefile.in	Sat Mar  1 11:49:19 2003
@@ -23,7 +23,7 @@
 
 ### simple, but effective rules
 
-all: dep do-all
+all: do-all
 
 dep:
 	touch src/$(DEPFILE) # to prevent unecessary warnings
@@ -58,7 +58,7 @@
 ### GENERIC
 
 # do-all goes to all subdirectories and does all
-do-%:
+do-%: dep
 	@target=`echo $@ | $(SED) -e 's/^do-//'`; \
 	for i in $(SUBDIRS) ; do \
 		if test -f ./$$i/Makefile ; then \
--- gpm-1.20.1/doc/Makefile.in	Sat Mar  1 11:56:51 2003
+++ gpm-1.20.1.new/doc/Makefile.in	Sat Mar  1 12:18:29 2003
@@ -57,10 +57,8 @@
 # MAN (-)
 $(MANPAGES): doc.gpm $(srcdir)/manpager
 	$(srcdir)/manpager doc.gpm
-	for i in gpm-root.1 gpm-types.7 gpm.8 mev.1 mouse-test.1; do \
-		expand $$i | sed s/^'[ ]*'//g > $$i.new; \
-		mv $$i.new $$i; \
-	done;
+	expand $@ | sed s/^'[ ]*'//g > $@.new
+	mv $@.new $@
 
 
 # DVI 
