--- ./doc/Makefile.in.org	Mon Mar 11 17:22:34 2002
+++ ./doc/Makefile.in	Mon Mar 11 17:24:58 2002
@@ -121,21 +121,8 @@
 	$(INSTALL_DATA) -m 644 gpm-root.1     $(DESTDIR)$(man1dir)
 	$(INSTALL_DATA) -m 644 gpm-types.7    $(DESTDIR)$(man7dir)
 	$(INSTALL_DATA) -m 644 gpm.8          $(DESTDIR)$(man8dir)
-	$(INSTALL_DATA) -m 644 $(srcdir)/gpm.info       $(DESTDIR)$(infodir)
+	$(INSTALL_DATA) -m 644 $(srcdir)/gpm.info*	$(DESTDIR)$(infodir)
 	# Use install-info if available
-	-if $(SHELL) -c 'install-info --version' >/dev/null 2>&1; then \
-	  if [ -f $(infodir)/dir ] ; then \
-		install-info --dir-file=$(DESTDIR)$(infodir)/dir $(DESTDIR)$(infodir)/gpm.info; \
-	  fi; \
-	fi
-	#itz Sat Dec 26 23:21:05 PST 1998
-	#i keep all my infopages compressed and i'm tired to do it by
-	#hand, so check if there are any compressed pages and do this
-	#one too
-	-ls $(DESTDIR)$(infodir)/*[-.]info.gz >/dev/null 2>&1 \
-		&& gzip -f $(DESTDIR)$(infodir)/gpm.info
-	# Hmm.... shouldn't man pages be compressed too?
-   # maybe they should, but at least at my system they are not.
 
 # This potentially leaves around cached manpages,
 # e.g. /var/cache/man/cat1/mev.1.gz.  Deleting these is not our job,
--- ./Makefile.include.in.org	Sat Feb 23 16:42:23 2002
+++ ./Makefile.include.in	Mon Mar 11 17:22:57 2002
@@ -45,7 +45,7 @@
 TEX = @TEX@
 TEXINDEX = @TEXINDEX@
 DVIPS = @DVIPS@
-MAKEINFO = @MAKEINFO@ --no-split
+MAKEINFO = @MAKEINFO@
 TEXI2HTML = texi2html -monolithic
 
 AR = ar
